pkg/synthetictests/duplicated_events.go

```diff
@@ -136,6 +136,9 @@ var allowedRepeatedEventPatterns = []*regexp.Regexp{
 
 	// Separated out in testMarketplaceStartupProbeFailure
 	regexp.MustCompile(marketplaceStartupProbeFailureRegExpStr),
+
+	// did not separate out. TODO: suppressing event introduced in 4.13(kube 1.26), look into undo this before GA
```

Comment:
Add a jira issue opened for appropriate team

Reply:
Opened https://issues.redhat.com/browse/NE-1163

Reply:
See https://issues.redhat.com/browse/OCPBUGS-5943.